src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -2599,6 +2599,13 @@ subroutine atm_divergence_damping_3d( state, diag, mesh, configs, dts, edgeStart
       rdts = 1.0_RKIND / dts
       coef_divdamp = 2.0_RKIND * smdiv * config_len_disp * rdts
 
+      MPAS_ACC_TIMER_START('atm_divergence_damping_3d [ACC_data_xfer]')
+      !$acc enter data copyin(ru_p, rtheta_pp, rtheta_pp_old, specZoneMaskEdge, &
+      !$acc         theta_m, cellsOnEdge) 
+      MPAS_ACC_TIMER_STOP('atm_divergence_damping_3d [ACC_data_xfer]')
+
+      !$acc parallel async
```

Comment:
`async` could be dangerous here, I don't believe `acc exit data copyout(ru_p)` will wait for this kernel to finish before beginning to copy.

I think it would be best to remove this `async` since we don't need it (yet) and it could create issues for the kernels that come next and depend on `ru_p`

Reply:
Will remove the `async`. thanks!

Reply:
I removed this async, but wanted to note that there's another one in `atm_advance_scalars_work`. Is that fine for now?